0-bootstrap/README.md

```diff
@@ -342,11 +351,160 @@ The following steps introduce the steps to deploy with Cloud Build Alternatively
 
 ## Running Terraform locally
 
-If you deploy using Cloud Build, the bucket information is replaced in the state
-backends as part of the build process when the build is executed by Cloud Build.
-If you want to execute Terraform locally, you need to add your Cloud
-Storage bucket to the `backend.tf` files.
-Each step has instructions for this change.
+The following steps introduce the steps to deploy without Cloud Build.
+
+1. Clone [terraform-example-foundation](https://github.com/terraform-google-modules/terraform-example-foundation) into your local environment and create to the `gcp-bootstrap` folder at the same level. Copy the `0-bootstrap` content and `.gitignore` to `gcp-bootstrap`.
+
+   ```bash
+   git clone https://github.com/terraform-google-modules/terraform-example-foundation.git
+
+   mkdir gcp-bootstrap
+
+   cp -R terraform-example-foundation/0-bootstrap/* gcp-bootstrap/
+
+   cp terraform-example-foundation/.gitignore gcp-bootstrap
+   ```
+
+1. Navigate to `gcp-bootstrap` and initialize a local git repository, so you can manage versions locally. Create the environment branches.
+
+   ```bash
+   cd gcp-bootstrap
+
+   git init
+   git checkout -b plan
+
+   git checkout -b shared
+   ```
+
+1. Rename `terraform.example.tfvars` to `terraform.tfvars` and update the file with values from your environment:
+
+   ```bash
+   mv terraform.example.tfvars terraform.tfvars
+   ```
+
+1. Rename `cb.tf` to `cb.tf.example` and update the file with values from your environment:
```

Comment:
If we are discarding the use of cb.tf then the text should not contain  the phrase "update the file with values from your environment". If that's not the case, ignore the comment